nasa-gibs/worldview
# WV-3117 Charting Feature Styling Updates

## Description
This change updates the styling of the Charting feature in the sidebar of Worldview to match the [wireframes](https://xd.adobe.com/view/1a2a1e24-9231-4d40-8a3c-9055ba9beda1-4f79/) and the [design comp](https://sketch.com/s/0f8e0b62-ff6a-4870-beea-f476f135faeb/a/1VmZ2Zn/play). The changes are mostly on the sidebar itself, although the charting crop interface and the charting graph have been updated as well.

## How To Test
1. `git checkout wv-3117-charting-wireframe-implementation`
2. `npm ci`
3. Modify line 7 in `features.json` to change `charting` from `false` to `true`
4. `npm run build`
5. `npm run watch`
6. Open a fresh instance of Worldview.
7. Verify that the default sidebar looks consistent with the designs, and that the Charting button is grayed-out.
8. Add a layer with a continuous color palette (Aerosol Index, Carbon Monoxide L3, etc.) and verify that the Charting button is no longer grayed-out.
9. Click the Charting button and verify that the Charting sidebar interface correctly reflects the designs.
10. Ensure that the functionality of the Charting interface still works properly (select date, change area of interest, change between one date and date range, etc.).
11. Add a layer with a non-continuous color palette (Deep Blue Aerosol Type) and verify that it does not appear in the list of layers during Charting.
12. Open the Comparison Mode and verify the sidebar interface correctly reflects the designs.
